api/specs.proto

```diff
@@ -160,8 +160,15 @@ message ContainerSpec {
 	string dir = 6;
 
 	// User specifies the user that should be employed to run the container.
+	//
+	// Note that the primary group may be specified by appending the group name
+	// or id to the user name, separated by a `:`. This syntax is
+	// `<user>:<group>`.
```

Comment:
Unfortunately, this is completely embedded down to libcontainer.

I've submitted https://github.com/docker/engine-api/pull/346, as well.